javascript/general: add general automated comments (#1140)
exercism/website-copy: automated-comments/javascript/general/no_named_export.md

No [export](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export)

> called `%<export.name>s`. The tests won't pass without it.

Reviewer: ```suggestion
called `%<export.name>s` was found. The tests won't pass without it.
```

> called `%<export.name>s` was found. The tests won't pass without it.

Author: ```suggestion
called `%{export.name}` was found. The tests won't pass without it.
```

> Make sure `%<export.name>s` exists and add `export` in front of it:

Reviewer 2: ```diff
- Make sure
+ Ensure
```

Author: ```suggestion
Ensure `%<export.name>s` exists and add `export` in front of it:
```

> Ensure `%<export.name>s` exists and add `export` in front of it:

Author: ```suggestion
Ensure `%{export.name}` exists and add `export` in front of it:
```

```javascript
// exporting a "const"

> export const %<export.name>s = /*...*/

Author: ```suggestion
export const %{export.name} = /*...*/
```

// exporting a named function declaration

> export function %<export.name>s() { /* ... */ }

Author: ```suggestion
export function %{export.name}() { /* ... */ }
```
